usecases/decision_usecase.go

```diff
@@ -549,9 +552,17 @@ func (usecase *DecisionUsecase) CreateAllDecisions(
 			usecase.scenarioEvaluator.EvalScenario(ctx, evaluationParameters)
 		switch {
 		case err != nil:
-			return nil, 0, errors.Wrap(err, "error evaluating scenario in CreateAllDecisions")
+			return nil, 0, errors.Wrapf(err, "error evaluating scenario \"%s\" in CreateAllDecisions", scenario.Name)
```

Comment:
Change the string to delimited by backticks instead, way more readable and less prone to escaping issues.